clients/spark/project/plugins.sbt

```diff
@@ -3,5 +3,4 @@ addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")
 addSbtPlugin("com.frugalmechanic" % "fm-sbt-s3-resolver" % "0.19.0")
 addSbtPlugin("cf.janga" % "sbts3" % "0.10.5")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.29")
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.0")
-addSbtPlugin("com.github.sbt" % "sbt-jacoco" % "3.3.0")
\ No newline at end of file
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.0")
```

Comment:
Can we please end files with a newline?  Most style guides recommend this; it sometimes matters when you want to run some script that does e.g.
```sh
echo 'new last line for file' >> clients/spark/project/plugins.sbt
```
because the expectation is that appending will be on a new line.